packages/components/src/ResourcePicker/Toolbar/StateFilter/StateFilter.scss

```diff
@@ -0,0 +1,57 @@
+$resource-picker-toolbar-favorite-color: $lightning-yellow !default;
+$resource-picker-toolbar-certified-color: $brand-primary !default;
+
+.state-filters {
+	margin-right: $padding-normal;
+
+	> * {
+		display: inline-block;
+		height: 100%;
```

Comment:
Same remark here, this makes the filter label block align weirdly compared to the other ones.
It's not really important because visually it's correct but, but what about removing that + flexbox on state-filters ?